moto/ses/responses.py

```diff
@@ -94,6 +94,35 @@ def send_templated_email(self):
         template = self.response_template(SEND_TEMPLATED_EMAIL_RESPONSE)
         return template.render(message=message)
 
+    def send_bulk_templated_email(self):
+        source = self.querystring.get("Source")[0]
+        template = self.querystring.get("Template")
+        template_data = self.querystring.get("DefaultTemplateData")
+
+        destinations = []
+        for i in range(1, 52):
+            destination_field = "Destinations.member.%s.Destination.ToAddresses.member.1" % (i)
+            if self.querystring.get(destination_field) is None:
+                break
+            destination = {"ToAddresses": [], "CcAddresses": [], "BccAddresses": []}
+            for dest_type in destination:
+                # consume up to 51 to allow exception
+                for j in range(1, 52):
+                    field = "Destinations.member.%s.Destination.%s.member.%s" % (i, dest_type, j)
+                    address = self.querystring.get(field)
+                    if address is None:
+                        break
+                    destination[dest_type].append(address[0])
+            destinations.append({"Destination": destination})
```

Comment:
Can we retrieve these parameters using one of the existing helper methods, such as `self._get_params()` or `self._get_multi_param_dict`?

Reply:
@bblommers
-  I tried using self._get_params(). There is `index out of bound` error when its used decode query string whose memebers are out of order (I know becuase my tests failed when i used them. but only on specific cases).
Note: I plan have look at this helper method later and fix it if possible
- Rest of the methods in this module uses the same convention I have. So for now can we keep it this way.